go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -3599,12 +3599,12 @@ var DoltTagTestScripts = []queries.ScriptTest{
 				Expected: []sql.Row{{types.OkResult{RowsAffected: 2}}},
 			},
 			{
-				Query:            "CALL DOLT_COMMIT('-am','made changes in other')",
-				SkipResultsCheck: true,
+				Query:    "CALL DOLT_COMMIT('-am','made changes in other')",
+				Expected: []sql.Row{{doltCommit}},
```

Comment:
This is so awesome to see us able to remove so many `SkipResultsCheck`s! 🙏 Great work!